src/csharp/Pester/Block.cs

```diff
@@ -40,7 +40,9 @@ public Block()
         public List<object> ErrorRecord { get; set; }
         public TimeSpan Duration { get => DiscoveryDuration + FrameworkDuration + UserDuration; }
 
-        public string Id { get; set; }
+        [Obsolete("Id is obsolete and should no longer be used. Use GroupId instead.")]
+        public string Id { get => GroupId; }
```

Comment:
Necessary? Obsolete-attribute isn't visible when invoked in PowerShell iirc.

Reply:
It's a good reminder for us internally. Keep.